QMLComponents/models/listmodellayersassigned.cpp

```diff
@@ -114,23 +104,23 @@ Terms ListModelLayersAssigned::addTerms(const Terms& terms, int dropItemIndex, c
 	
 	beginResetModel();
 	
-	int layer = _variables.length();
+	int layer = _variablesPerLayer.length();
 	int indexInLayer = 0;
 	if (dropItemIndex >= 0)
-		layer = _getLayer(dropItemIndex, indexInLayer, true);
+		std::tie(layer, indexInLayer) = _getLayer(dropItemIndex, true);
```

Comment:
```suggestion
		auto [layer, indexInLayer] = _getLayer(dropItemIndex, true);
```

In other places you already use structured bindings right?
Why use std::tie in only two occasions then? 
I had to look up what it does only to find: https://stackoverflow.com/a/43780252